toolkit/tools/pkggen/worker/create_worker_chroot.sh

```diff
@@ -100,3 +124,6 @@ else
     tar -I gzip -cvf "$chroot_archive" -C "$chroot_base/$chroot_name" . >> "$chroot_log"
 fi
 echo "Done creating $chroot_archive." | tee -a "$chroot_log"
+
+record_timestamp "Done installing all packages"
```

Comment:
We've finished installing all packages a few lines above. Here we've also packed them and that takes a while. I'd split this into two steps.